automated/android/noninteractive-tradefed/monitor_fastboot.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/sh -x
 while true; do
```

Comment:
Not this patch related, but looks like the script simply run 'fastboot boot'  every 30 seconds. I think we should at least check if a fastboot device appeared.

Reply:
fastboot will wait indefinitely for the device to appear. The whole idea is to boot the board if it goes to fastboot mode.